Resources/about.json

```diff
@@ -0,0 +1,20 @@
+{
+	"appInfo": {
+		"appName": "SwiftIsland",
+		"appVersion": "1.1.1",
```

Comment:
What is the meaning of `appVersion`? The version the user has installed or the latest version available. If the latter, we should rename it to `latestAppVersion`; if the former then this should be done by the app itself since the API won't know which version the client is using.

Reply:
oh yes you are totally right! 😄 
Maybe latestAppVersion is better, so that users can see their vs. the most recent version